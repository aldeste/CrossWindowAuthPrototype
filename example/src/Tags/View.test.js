import View from "./View";

describe("View tag", () => {
  it("Should be defined", () => {
    expect(View).toBeDefined();
  });
});
