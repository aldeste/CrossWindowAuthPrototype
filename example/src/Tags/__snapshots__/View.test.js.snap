// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View tag Matches earlier configuration 1`] = `


.fIwvJd {
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  border: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column;
  -ms-flex-flow: column;
  flex-flow: column;
  min-height: : 0,min-width: 0;
}

<div
  className="View__View-jelnfZ fIwvJd"
/>
`;
